ui/src/app/core/settings.service.ts

```diff
@@ -52,6 +55,10 @@ export class SettingsService {
     return this.getSettingsForProject(projectId).pageSize;
   }
 
+  getHideArchived(projectId: string): boolean {
+    return this.getSettingsForProject(projectId).hideArchived;
+  }
+
   /** update the display columns settings for a project */
```

Comment:
nit: I'd remove this comment even though you didn't add it. It's pretty clear what this function is doing and the other similar ones don't have comments.

Reply:
k, done